.github/workflows/support-workers-build.yml

```diff
@@ -36,12 +36,18 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           cache: "yarn"
-          cache-dependency-path: cdk/yarn.lock
+          cache-dependency-path: '**/yarn.lock'
 
       - name: Build CFN from CDK
         run: ./script/ci
         working-directory: cdk
 
+      - name: Build TypeScript lambdas
+        run: |
+          yarn install --frozen-lockfile
```

Comment:
Should we be using `pnpm` for a new TypeScript project?

Reply:
Yes good idea